editor/plugins/multimesh_editor_plugin.cpp

```diff
@@ -171,8 +171,9 @@ void MultiMeshEditor::_populate() {
 
 	int instance_count = populate_amount->get_value();
 
-	multimesh->set_transform_format(MultiMesh::TRANSFORM_3D);
-	multimesh->set_color_format(MultiMesh::COLOR_NONE);
+	multimesh->set_transform_format(node->get_multimesh()->get_transform_format());
```

Comment:
Is there ever a situation where "populate" should be allowed to use a 2D transform?

Reply:
No, you're right. I see that a `Transform` is set to the instances later in the method so `transform_format` should not be changed. I was confused by the option to choose **2D** as the _Transform Format_ for the `MultiMesh` in the `MultiMeshInstance`. I'll change it back as soon as possible.